fix : address 엔티티,유저생성 responseDto 수정.

wls313 opened the pull request:
address 에 빈값이 들어갈수 없도록 address 에 nullable = false 를 추가했습니다.

유저생성 responseDto 에 들어있던 password 를 제거했습니다.

Commit:
fix : address 엔티티,유저생성 responseDto 수정.

address 에 빈값이 들어갈수 없도록 address 에 nullable = false 를 추가했습니다.

유저생성 responseDto 에 들어있던 password 를 제거했습니다.

Developer-Groo commented on src/main/java/com/example/Sparta_Store/user/dto/CreateUserResponseDto.java:
```diff
@@ -5,14 +5,12 @@
     String name,
     Address address
 ) {
     public static CreateUserResponseDto createDto(User user) {
```
Dto 생성 메서드 이름은 toDto() 로 통일 부탁드립니다!

Commit:
fix : jwt 위치 이동,유저생성 dto 수정

Dto 생성 매서드 이름 변경,

Commit:
fix : jwt 위치 이동,유저생성 dto 수정

Dto 생성 매서드 이름 변경,

Commit:
commit

Commit:
fix : ROLE -> NAME

ROLE 을 따로 만드는것보다 name 으로 체크하는게 나을거같아서 name 으로 변경했습니다. 이에따라 회원가입시 이름을 ADMIN 으로 할수없도록 변경했습니다.
